src/DeviceInterface/AppleOverlayDeviceInterface.js

```diff
@@ -45,6 +45,11 @@ class AppleOverlayDeviceInterface extends AppleDeviceInterface {
         })
     }
 
+    async startCredentialsImportFlow () {
+        this._closeAutofillParent()
+        await this.deviceApi.notify(createNotification('startCredentialsImportFlow'))
+    }
+
```

Comment:
Shouldn't this method be in the `CredentialsImport` class?

Reply:
Now that windows/mac both with keep the promise open (originally I wasn't sure if that's possible), I have moved it to the `CredentialsImport` class :)